### PR TITLE
net/can: Remove the unnecessary "ret = OK;" in can_getsockopt

### PR DESCRIPTION
**Pull request by xiaoxiang781216**
## Summary

and remove the extra space in code

## Impact

code refactor only

## Testing

CI

### CODE REVIEW

File: net/can/can_getsockopt.c [hunk 1]
```diff
@@ -172,14 +169,13 @@ int can_getsockopt(FAR struct socket *psock, int level, int option,
              * to me in this case.
              */
 
-            ret              = -EINVAL;
+            ret  = -EINVAL;
```
Comment (pkarashchenko) [inline net/can/can_getsockopt.c:172]:
```suggestion
            ret = -EINVAL;
```

File: net/bluetooth/bluetooth_container.c [hunk 2]
```diff
@@ -144,7 +144,7 @@ FAR struct bluetooth_container_s *bluetooth_container_allocate(void)
       net_unlock();
       container = (FAR struct bluetooth_container_s *)
         kmm_malloc((sizeof (struct bluetooth_container_s)));
```
Comment (pkarashchenko) [inline net/bluetooth/bluetooth_container.c:146]:
```suggestion
        kmm_malloc((sizeof(struct bluetooth_container_s)));
```

Comment (xiaoxiang781216) [reply net/bluetooth/bluetooth_container.c]: Done.

File: net/can/can_getsockopt.c (hunk 1 above)
Comment (xiaoxiang781216) [reply net/can/can_getsockopt.c]: Done.

**Review by jerpelea — APPROVED** [review]

**Review by jerpelea — APPROVED** [review]

**Review by pkarashchenko — APPROVED** [review]

**Pull request merged**